templates/single-tapestry.php

```diff
@@ -41,17 +43,25 @@ function addNonceToScript()
                 comments_template();
             endif;
             ?>
+        
+            <?php endwhile; // end of the loop. ?>
 
-        <?php endwhile;
-    ?>
+        <div id="tapestry">
+            <!-- Don't render add root button when there's nodes -->
+            <?php if(!doesTapestryHaveNodes(get_the_ID())) : ?>
+                <div id="root-node-btn"><i class="fas fa-plus-circle fa-5x"></i></div>
```

Comment:
Could you please change this to have the text "Add Root Node" either inside the button or beside it (whatever looks good). I just want to highlight it more for new users.

Reply:
done